test/data/oauth-token-signer.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Move cryptomaterial generation into fixtures to avoid alerts being fired by sensitive material checking.

Reply:
done